src/netex_idf/read.rs

```diff
@@ -45,6 +47,21 @@ where
     stops::from_path(&path.join("arrets.xml"), &mut collections)?;
     // TODO : use _lines_netex_idf to get trips>physical_mode_id
     let _lines_netex_idf = lines::from_path(&path.join("lignes.xml"), &mut collections)?;
+    for offer_folder in WalkDir::new(path)
+        .min_depth(1)
+        .max_depth(1)
+        .into_iter()
+        .filter_map(std::result::Result::ok)
```

Comment:
proposal
.filter_map(|entry| entry.ok())
.filter(|entry| entry.file_type().is_dir())

Reply:
Nice, I was not a fan of this `.metadata()` API but it seems I completely missed the `.file_type()` one. Fixed in 3ebad02c1150254f0a168007a0c4ab44fc255ca6.